src/main/java/com/sphy/hotelmanagementapplication/security/JwtUtil.java

```diff
@@ -0,0 +1,95 @@
+package com.sphy.hotelmanagementapplication.security;
+
+import com.sphy.hotelmanagementapplication.service.UserService;
+import io.jsonwebtoken.Claims;
+import io.jsonwebtoken.Jwts;
+import io.jsonwebtoken.SignatureAlgorithm;
+import org.springframework.security.core.userdetails.UserDetails;
+import org.springframework.stereotype.Service;
+
+import java.util.Date;
+import java.util.HashMap;
+import java.util.Map;
+import java.util.function.Function;
+
+/***
+ * created by gp
+ */
+@Service
+public class JwtUtil {
+
+    private String SECRET_KEY = "mcbvhadfsjbmnwrctsfzjhd";
```

Comment:
Please move this in     
@Value("${SECRET_KEY}")
private String SECRET_KEY;
and DO NOT ADD the value to application.properties